packages/react-microlink/src/utils/styleResets.js

```diff
@@ -0,0 +1,10 @@
+import { css } from 'styled-components'
+
+export const backgroundColorReset = css`
+  background-color: inherit;
+`
+
+export const sizeReset = css`
```

Comment:
I was thinking more in a file that content reset per each component, for example:

```jsx
export const cardDescription = css`
text-decoration: none;
`
```

Then when you load the specific reset style into the component is easy to determinate what style are part of the component and what are part of the reset:



```
${reset.cardDescription}; // resets!
// specific styles
font-size: 14px;
flex-grow: 2;
margin: auto 0;
line-height: 18px;
${({cardSize}) => !isLarge(cardSize) && mobileDescriptionStyle};
```